conductr_cli/sandbox_common.py

```diff
@@ -14,6 +10,23 @@ class ConductrComponent(Enum):
                   9006}  # ConductR bundleStreamServer
 CONDUCTR_DEV_IMAGE = 'typesafe-docker-registry-for-subscribers-only.bintray.io/conductr/conductr'
 LATEST_CONDUCTR_VERSION = '1.1.11'
```

Comment:
We should move this into `sandbox_run_docker`, or perhaps remove it altogether since it's not used anywhere but in test.

I'm happy to do this in my own PR.